cli/task_queue.go

```diff
@@ -54,7 +57,7 @@ func newTaskQueueCommands() []*cli.Command {
 		},
 		{
 			Name:  "list-partition",
-			Usage: "List all the taskqueue partitions and the hostname for partitions",
+			Usage: "List the Task Queue's partitions and their hostnames",
```

Comment:
```suggestion
			Usage: "List the Task Queue's partitions and which matching node they are assigned to",
```
btw, this is going to go away at some point. partitions should not be exposed to the user (at this level)